$NetBSD: patch-configure,v 1.1 2013/10/07 12:05:46 makoto Exp $

ERROR: [check-portability.awk] => Found test ... == ...:
ERROR: [check-portability.awk] configure:               if test "$py_version" == "None"; then

--- configure~	2013-10-07 18:29:26.000000000 +0900
+++ configure	2013-10-07 18:30:41.000000000 +0900
@@ -22774,7 +22774,7 @@ echo $ECHO_N "checking for Python librar
 		py_version=`$PYTHON -c "from distutils.sysconfig import *; \
 			from string import join; \
 			print join(get_config_vars('VERSION'))"`
-		if test "$py_version" == "None"; then
+		if test "$py_version" = "None"; then
 			if test -n "$PYTHON_VERSION"; then
 				py_version=$PYTHON_VERSION
 			else
